apps/graphql/src/queries/__tests__/CustomerBookings.test.js

```diff
@@ -0,0 +1,9 @@
+// @flow
```

Comment:
Martín happy. 👏 The test file shouldn't be named like this though. It implies it's only for `customerBookings` query and people tend to copy-paste this test later on. It's better to do something like `queries.test.js` because these five lines can test every query imaginable in fact (just create new fixture file). No need to write this JS test ever again. That's the whole point of this utility... :)

Reply:
I will rename it